cmd/kola/options.go

```diff
@@ -167,7 +167,7 @@ func syncOptions() error {
 	}
 
 	image, ok := kolaDefaultImages[kola.QEMUOptions.Board]
-	if !ok {
+	if kola.QEMUOptions.Distribution == "cl" && !ok {
```

Comment:
So this is unsupported only if it's Container Linux *and* the `kolaDefaultImages` doesn't have an entry?

Reply:
The [`kolaDefaultImages` variable](https://github.com/coreos/mantle/blob/master/cmd/kola/options.go#L35-L38) is a hashmap of where the images would live inside of the CL SDK, it's not really relevant outside of the context of CL. We could implement a secondary check for unsupported boards on other platforms but I suggested that the check just be modified in a comment [here](https://github.com/coreos/mantle/pull/1048#discussion_r317183787).